src/main/java/org/apache/commons/collections4/CollectionUtils.java

```diff
@@ -1791,6 +1791,65 @@ public static int maxSize(final Collection<? extends Object> coll) {
         return list;
     }
 
+    /**
+     * Removes elements whose index are between startIndex and endIndex in the collection and returns them.
+     * This method modifies the input collections.
+     *
+     * @param <E>  the type of object the {@link Collection} contains
+     * @param input  the collection will be operated, can't be null
+     * @param startIndex  the start index to remove element, can't be less than 0
+     * @param endIndex  the end index to remove, can't be less than startIndex
```

Comment:
The Javadoc for indexes should specify whether they are inclusive or exclusive, for example, see https://docs.oracle.com/javase/8/docs/api/java/util/List.html#subList-int-int-